html5/js/Client.js

```diff
@@ -326,7 +326,7 @@ class XpraClient {
     const screen_element = jQuery("#screen");
     screen_element.mousedown((e) => this.on_mousedown(e));
     screen_element.mouseup((e) => this.on_mouseup(e));
-    screen_element.mousemove((e) => this.on_mousemove(e));
+    document.getElementById("screen").addEventListener("mousemove", (e) => this.on_mousemove(e));
```

Comment:
Why change this one and not the ones above it? 
What's the difference between them?

Reply:
jQuery is too old, and does not support the movementX and movementY event variables that are needed for cursor lock